src/components/sections/contest/ContestDetails/ContestInfo.tsx

```diff
@@ -115,54 +136,78 @@ const ContestInfo = ({
         <p className="text-bodyM">{contest.customConditions || '-'}</p>
       </div>
 
-      {session?.user.role === UserRole.JUDGE &&
-        contest.status === ContestStatus.IN_REVIEW && (
-          <div className="flex gap-8">
-            <AlertDialog>
-              <AlertDialogTrigger>
-                <Button variant="outline" className="flex gap-3">
-                  <span className="uppercase">Approve contest</span>
-                </Button>
-              </AlertDialogTrigger>
-
-              <AlertDialogContent>
-                <AlertDialogHeader>
-                  <AlertDialogTitle className="uppercase">
-                    Are you sure you want to approve this contest?
-                  </AlertDialogTitle>
-                </AlertDialogHeader>
-                <AlertDialogFooter>
-                  <AlertDialogCancel>Cancel</AlertDialogCancel>
-                  <AlertDialogAction onClick={approveContest}>
-                    Yes, approve
-                  </AlertDialogAction>
-                </AlertDialogFooter>
-              </AlertDialogContent>
-            </AlertDialog>
-
-            <AlertDialog>
-              <AlertDialogTrigger>
-                <Button variant="outline" className="flex gap-3">
-                  <span className="uppercase">Reject contest</span>
-                </Button>
-              </AlertDialogTrigger>
-
-              <AlertDialogContent>
-                <AlertDialogHeader>
-                  <AlertDialogTitle className="uppercase">
-                    Are you sure you want to reject this contest?
-                  </AlertDialogTitle>
-                </AlertDialogHeader>
-                <AlertDialogFooter>
-                  <AlertDialogCancel>Cancel</AlertDialogCancel>
-                  <AlertDialogAction onClick={rejectContest}>
-                    Yes, reject
-                  </AlertDialogAction>
-                </AlertDialogFooter>
-              </AlertDialogContent>
-            </AlertDialog>
-          </div>
+      <div className="mb-12 flex flex-col gap-3">
+        <h3 className="text-bodyL text-purple-light">
+          Rewards Transfer TX Hash
+        </h3>
+        {contest.rewardsTransferTxHash ? (
+          <Link
+            href={`https://cardanoscan.io/transaction/${contest.rewardsTransferTxHash}`}
+            className="text-bodyM underline">
+            {contest.rewardsTransferTxHash}
+          </Link>
```

Comment:
Wouldn't it be better to wrap it in `<Button variant="link">` to use the same design for links as everywhere?

Reply:
Sure 👍